android/src/app/organicmaps/help/HelpFragment.java

```diff
@@ -142,11 +142,17 @@ else if (id == R.id.faq)
     else if (id == R.id.report)
       Utils.sendBugReport(requireActivity(), "");
     else if (id == R.id.support_us)
-      openLink(Constants.Url.SUPPORT_US);
+      openLink(getResources().getString(R.string.translated_om_site_url) + "support-us/");
     else if (id == R.id.donate)
       openLink(mDonateUrl);
     else if (id == R.id.rate)
-      Utils.openAppInMarket(requireActivity(), BuildConfig.REVIEW_URL);
+    {
+      // Some flavors don't have review url and need to use localized donate links.
+      if (BuildConfig.REVIEW_URL.startsWith("https://organicmaps.app/donate/"))
```

Comment:
Probably it would be better to set REVIEW_URL to "" in build.gradle.

Reply:
My implementation is safer. Let's leave it as is.

Reply:
Sorry, but this is a hideous hack. This URL is hard-coded in one place to be compared with hard-coded string in another place... We shouldn't have this "Rate the app" button at all for web/fdroid because it doesn't make sense at all.